server/src/test/java/org/elasticsearch/transport/TransportHandshakerTests.java

```diff
@@ -67,7 +67,7 @@ public void testHandshakeRequestAndResponse() throws IOException {
         long reqId = randomLongBetween(1, 10);
         handshaker.sendHandshake(reqId, node, channel, new TimeValue(30, TimeUnit.SECONDS), versionFuture);
 
-        verify(requestSender).sendRequest(node, channel, reqId, TransportVersion.CURRENT.minimumCompatibilityVersion());
+        verify(requestSender).sendRequest(node, channel, reqId, TransportHandshaker.EARLIEST_HANDSHAKE_VERSION);
```

Comment:
This change (and the other similar ones in this suite) moves the handshake version used from 7.17.0 to 6.8.0 which I think is not what we want. Indeed we should be able to assert somewhere that we only ever send 7.17.0-format handshake requests (in prod at least, we could reasonably have some kind of test utility to emulate the way that 6.8.0 handshake requests are created)